README.md

```diff
@@ -35,6 +41,91 @@ To see the graphics, you can open another terminal and run
+
+### [Simple Navigation](python/social_bot/envs/simple_navigation.py)
+A simple navigation task for a pioneer2dx agent.
+
+* [Goal task](python/social_bot/teacher_tasks.py). 
+    
+    Gif to be updated
+
+### [Grocery Ground](python/social_bot/envs/grocery_ground.py)
+A playground with groceries on it. You could choose agent in the environment by setting the paramenter "agent_type". We support pioneer2dx, pr2, turtlebot, irobot create, and icub for now. 
+[ICub](http://www.icub.org) is an humanoid robot meant for more complex tasks in the future. You could also choose icub_with_hands, which is a more advanced version of icub that equipped with both cameras and dexterous hands.
+
+* [Goal task](python/social_bot/envs/grocery_ground.py): A task to chase a goal. 
+
+    <img src="media/grocery_ground_pioneer.gif" width="320" height="240" alt="pioneer"/> <img src="media/grocery_ground_icub.gif" width="360" height="240" alt="icub"/>
+
+    The agent will receive reward 1 when it is close enough to the goal, and get -1 if moving away from the goal too much or timeout.
+
+### [PR2 Gripping](python/social_bot/envs/pr2.py)
+
+* A task that the agent need to use its grippers or fingers to grip a beer.
+
+    Gif To be updated
+
+    A simple reward shaping is used to guide the agent's gripper to get close to the target, open the gripper and lift the target up.
+
+### [iCub Walking](python/social_bot/envs/icub_walk.py)
+
+* A simple humainoid walking task.
+
+    <img src="media/icub_walk.gif" width="320" height="240" alt="pioneer"/> 
+    
+    reward = not_fall_bonus + truncked_walk_velocity - ctrl_cost
+ 
+
+## Training Examples
+### [Simple Navigation Training Example](https://github.com/HorizonRobotics/alf/blob/master/alf/examples/ac_simple_navigation.gin)
+Training with [Agent Learning Framework (Alf)](https://github.com/HorizonRobotics/alf)
+```bash
+cd ALF_REPO_ROOT/examples/
+python -m alf.bin.main --root_dir=~/tmp/simple_navigation --gin_file=ac_simple_navigation.gin --alsologtostderr
+```
+### [PR2 Training Example](https://github.com/HorizonRobotics/alf/blob/master/alf/examples/ppo_pr2.gin)
+Training with Alf:
+```bash
+python -m alf.bin.main --root_dir=~/tmp/pr2_gripping --gin_file=ALF_REPO_ROOT/examples/ppo_pr2.gin --alsologtostderr
+```
+
+### [Grocery Ground Training Example](examples/grocery_alf_ppo.gin)
+Training with Alf:
+```bash
+python -m alf.bin.main --root_dir=~/tmp/grocery_ppo --gin_file=grocery_alf_ppo.gin --alsologtostderr
+```
+
+### [iCub Walking Task Training Example](examples/icub_alf_ppo.gin)
+Training with Alf:
+```bash
+python -m alf.bin.main --root_dir=~/tmp/icub_ppo --gin_file=icub_alf_ppo.gin --alsologtostderr
+```
+Or you can try with tf-agent SAC, which might be better at sample efficiency:
+```bash
+python examples/train_icub_walk_sac.py \
+    --root_dir=~/tmp/ICubWalkSAC
+```
 
```

Comment:
Maybe add a section on how to play the trained model using --play.

Reply:
Added.